web/turkey/survey/static/survey/js/auditors/clicks_specific.js

```diff
@@ -0,0 +1,33 @@
+var AuditorClicksSpecifics = {
+    clicks_specifics: [],
+    log_click_content: function (e) {
+        var dom = {
+            'type'  :   e.target.nodeName.toLowerCase(),
+            'id'    :   e.target.id != '' && e.target.id != undefined 
```

Comment:
I'd keep it null. Making fields nullable is as simple as setting kwarg `null=True` in a field constructor. And it will be faster if we decide to aggregate on nulls in any way.